src/AddedTasks.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import './AddedTasks.css';
+import deletePng from './images/delete.png';
+import editPng from './images/edit.png';
+
+export default class AddedTasks extends Component {
+  render() {
+    const tasks = this.props.tasks;
+    
+    let tasksElements = tasks.map(task => {
+      return (
+        <div className="added-task_container" key={task.id} id={task.id}>
```

Comment:
better to create <Task> component

Reply:
Sergey, please tell me what it is for, I cannot understand how to do it.